Archipelago.HollowKnight/MC/ArchipelagoModeMenuConstructor.cs

```diff
@@ -153,7 +153,9 @@ private static void StartOrResumeGame(bool newGame, MenuLabel errorLabel)
             }
             else
             {
-                Archipelago.Instance.MenuSettings = Archipelago.Instance.ApSettings with { };
+                Archipelago.Instance.MenuSettings = Archipelago.Instance.ApSettings with {
+                    AlwaysShowItems = Archipelago.Instance.MenuSettings.AlwaysShowItems
```

Comment:
I don't think this should be needed

Reply:
Unfortunately it is. Without that change the save-specific ApSettings will override the MenuSettings whenever a save file is loaded, which causes the `AlwaysShowItems` setting to be overridden. If you have another solution I'm happy to hear it, but this was the simplest fix I was able to find.

Reply:
Oh yuck didn't think of that. Settings really need to be refactored but there is just not a way to do it that doesn't break anyone with an existing install

Reply:
I'm sure it's possible to do while still being backwards-compatible, but it was a bigger change than I wanted to make in this PR, so I opted for the simpler solution. I'm happy to take a crack at that refactor if you'd prefer it done that way though.